test/components/DayPickerRangeController_spec.jsx

```diff
@@ -796,6 +796,142 @@ describe('DayPickerRangeController', () => {
         });
       });
 
+      describe('blocked', () => {
+        describe('focusedInput did not change', () => {
+          it('does not call isBlocked', () => {
+            const isBlockedStub =
+              sinon.stub(DayPickerRangeController.prototype, 'isBlocked');
+            const wrapper = shallow(<DayPickerRangeController {...props} />);
+            isBlockedStub.reset();
+            wrapper.instance().componentWillReceiveProps({
+              ...props,
+            });
+            expect(isBlockedStub.callCount).to.equal(0);
+          });
+        });
+
+        describe('focusedInput changed', () => {
+          const visibleDays = {
+            [toISOMonthString(today)]: {
+              [toISODateString(today)]: [],
+              [toISODateString(moment().add(1, 'day'))]: [],
+              [toISODateString(moment().add(2, 'day'))]: [],
+            },
+          };
+          const numVisibleDays = 3;
+
+          it('calls isBlocked for every visible day', () => {
+            const isBlockedStub =
+              sinon.stub(DayPickerRangeController.prototype, 'isBlocked');
+            const wrapper = shallow(<DayPickerRangeController {...props} />);
+            wrapper.setState({ visibleDays });
+            isBlockedStub.reset();
+            wrapper.instance().componentWillReceiveProps({
+              ...props,
+              focusedInput: END_DATE,
+            });
+            expect(isBlockedStub.callCount).to.equal(numVisibleDays);
+          });
+
+          it('if isBlocked(day) is true calls addModifier with `blocked` for each day', () => {
+            const addModifierSpy = sinon.spy(DayPickerRangeController.prototype, 'addModifier');
+            const isBlockedStub =
+              sinon.stub(DayPickerRangeController.prototype, 'isBlocked').returns(true);
+            const wrapper = shallow(<DayPickerRangeController {...props} />);
+            wrapper.setState({ visibleDays });
+            wrapper.instance().componentWillReceiveProps({
+              ...props,
+              focusedInput: START_DATE,
+            });
+            const blockedCalendarCalls = getCallsByModifier(addModifierSpy, 'blocked');
+            expect(blockedCalendarCalls.length).to.equal(numVisibleDays);
+            isBlockedStub.restore();
```

Comment:
@majapw thanks for the tip re: resetting stubs! However, in this test and the following one, I need to make a stub that returns `true` or `false`. However, I don't ever need to access the stub, as `blockedCalendarCalls` is what's used to verify the test.

I realized I was mistakenly passing the stub in with the props in my previous commit (doing nothing) but after removing it I started getting a `no-unused-vars` lint error. So I shoehorned `isBlockedStub.restore()` in. Would this be an acceptable place for `// eslint-ignore-line no-unused-vars`?

Thanks!